menpo/shape/pointcloud.py

```diff
@@ -379,11 +379,63 @@ def _from_vector_inplace(self, vector):
         """
         self.points = vector.reshape([-1, self.n_dims])
 
+    def same_space(self, other):
```

Comment:
`same_shape` a better name? Also does this really save keystrokes from `p1.shape == p2.shape` which is very self explanatory?

Reply:
I think you are right. I will remove it.